.github/workflows/execute_notebooks.yml

```
@@ -0,0 +1,91 @@
+            --ExecutePreprocessor.kernel_name=python3
+          
+          # Check if execution was successful
+          if [ $? -eq 0 ]; then
+            echo "✓ Successfully executed: $notebook"
+          else
+            echo "✗ Failed to execute: $notebook"
+            exit 1
+          fi
+        done < notebooks_to_execute.txt
+    - name: Push changes
+      if: steps.verify-changed-files.outputs.changed == 'true' && github.event_name != 'pull_request'
```

actionlint (1.7.7)

property "verify-changed-files" is not defined in object type {}

(expression)